gini/gini-vision-lib-ios
# Add images to analysis result callback

### Description
When getting the analysis result, now the images processed are also attached.

### How to test
Run the example app and check that the images are returned in the delegate method.

### Merging
Automatic

### Issues
